spec/jobs/bookmark_reminder_notifications_spec.rb

```diff
@@ -35,10 +35,10 @@
 
   it "will not send a reminder for a bookmark in the future" do
     bookmark4 = Fabricate(:bookmark, reminder_at: Time.zone.now + 1.day)
-    BookmarkReminderNotificationHandler.expects(:send_notification).with(bookmark1)
-    BookmarkReminderNotificationHandler.expects(:send_notification).with(bookmark2)
-    BookmarkReminderNotificationHandler.expects(:send_notification).with(bookmark3)
-    BookmarkReminderNotificationHandler.expects(:send_notification).with(bookmark4).never
+    BookmarkReminderNotificationHandler.any_instance.expects(:send_notification)
+    BookmarkReminderNotificationHandler.any_instance.expects(:send_notification)
+    BookmarkReminderNotificationHandler.any_instance.expects(:send_notification)
```

Comment:
The change here has made the assertion less specific so I'm not sure if that is a good thing. On another note, is there a way we can remove the mocks here?

Reply:
Hmm yeah I think I can just check for the created notification based on the bookmark details, will change.